content/docs/design/visual_story/congratulations@ko.md

```diff
@@ -0,0 +1,25 @@
+---
+$title: 축하합니다!
+---
+
+AMP 스토리를 완성했습니다!
+
+본 자습서를 통해 배운 것들은 다음과 같습니다:
+
+- 다수의 페이지로 구성된 [AMP 스토리](/ko/docs/reference/components/amp-story.html) 컴포넌트를 만들어보았습니다.
+- 다수의 레이어로 구성된 페이지를 만들어보았습니다.
+- 여러가지 템플릿으로 요소를 배치해보았습니다.
+- 페이지에 머무는 동안 재생될 배경 음악을 추가해보았습니다.
+- 페이지의 요소에 애니메이션 효과를 적용해보았습니다.
+- 독자의 관심을 끌기 위해 스토리의 마지막에 연관된 다른 콘텐츠로의 링크를 추가해보았습니다.
+
+## 다음 단계
+
+AMP 스토리에 대한 좀 더 자세한 자료는 다음과 같습니다:
+
+- [amp-story 레퍼런스 문서](/ko/docs/reference/components/amp-story.html)
+- [AMP 스토리 베스트 프랙티스](/ko/docs/design/amp_story_best_practices.html)
+
+{% call callout('Note', type='note') %}
```

Comment:
Can you translate "Note"?  (The 1st string, the second one is a type variable that must remain as is.)